returnn/tf/network.py

```diff
@@ -823,7 +823,6 @@ def get_layer(src_name):
         "layer %r not found in %r" % (name, self), layer_name=full_name, network=self, net_dict=net_dict)
     if not add_layer:
       add_layer = self.add_layer
-
```

Comment:
Try to avoid unrelated changes.

Reply:
was testing something in that line and forgot that it was a blank line before